spec/portal/fixtures/download_certificate_failure.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
Huh, yeah, had the same thought. Will nuke this file from history. `push -f` incoming.